### DISCUSSION TITLE
Password problem

### DISCUSSION THREAD

**Discussion by vxvr** [2fast-team/2fast#51] [Q&A] [answered]
Hello, this is more of a support ticket than a bug report, but basically i tried to link my 2fa to one of my accounts. it worked well for a week or so but i logged out, and the password seems to have reset. ive been trying for atleast a month now to get back into my 2fa file but i cant. is there anyway you could help me? or maybe reset the password?

**Comment by jp-weber** [comment, marked as answer]
Hello @vxvr , you did not change a password on your own? Do you have the file in a cloud storage (nextcloud, google, microsoft, etc)? In a cloud storage, you could switch to an older version of the file. 

What steps have you tested so far to log in? The current password for the app can be found in the windows settings with the name Project2FA:
https://support.microsoft.com/en-us/windows/accessing-credential-manager-1b5c916a-6a16-889f-8581-fc16e8165ac0

**Reply by vxvr** [reply]
thank you so much! i didnt know that was a feature.

**Comment by franco-bez** [comment]
I had a similar Problem already twice.
first time some 2 month ago, last time yesterday.
First time I installed an update of the 2fast app and afterwards my password did not get accepted any longer. 
I got asked to change the password in a Dialog with old password new password fields, but no matter what I tried i was locked out.

Strange enough, the next day everthing worked as usual, with the same file.

Yesterday I'm not sure whether there was an update of the app or not, but for sure some new Windows Updates.
Otherwise the same story, I could not use the app yesterday as my password did not get accepted, today erverthing works fine again.

**Reply by jp-weber** [reply]
A long time ago, version 1.2.4 had problems with older data files, in which a dialogue with a change of password was displayed, but this had no function on the file. The problem was promptly solved by version 1.2.5. 

However, I can't understand why the password can't be entered at the moment and then a short time later for you with the application. is it possible that the keyboard layout has changed in the language? The password is not copied or entered through a password manager?

**Reply by franco-bez** [reply]
The keyboard layout did not change, and the password consists only of letters that are on the same key in the english qwerty, as well as in the german qwertz layout. Also the password is entered manually, no password manager.

It didn't accept any password at all, and it repeated to ask for a password change just for ever. Even an uninstallation and reinstallation of the app did not help.

As I stated above, the next day I tried again and everything worked again as if nothing had happened.
